modules/backup-etcd.adoc

```diff
@@ -14,26 +14,37 @@ Only save a backup from a single master host. Do not take a backup from each mas
 
 .Prerequisites
 
-* SSH access to a master host.
-
-.Procedure
-
-. Access a master host.
-
-. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have checked whether the cluster-wide proxy is enabled.
 +
 [TIP]
 ====
 You can check whether the proxy is enabled by reviewing the output of `oc get proxy cluster -o yaml`. The proxy is enabled if the `httpProxy`, `httpsProxy`, and `noProxy` fields have values set.
 ====
 
-. Run the `cluster-backup.sh` script and pass in the location to save the backup to.
+.Procedure
+
+. Start a debug session for a master node:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+. Change your root directory to the host:
 +
-Be sure to pass in the `-E` flag to `sudo` so that environment variables are properly passed to the script.
+[source,terminal]
+----
+sh-4.2# chroot /host
+----
+
+. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+
+. Run the `cluster-backup.sh` script and pass in the location to save the backup to.
 +
 [source,terminal]
 ----
-$ sudo -E /usr/local/bin/cluster-backup.sh ./assets/backup
+sh-4.4# /usr/local/bin/cluster-backup.sh /home/core/assets/backup
```

Comment:
Is `/home/core/assets/backup` required as the backup destination?

Reply:
No, just an example.